### PR TITLE
Adding NFR questionnaire

### PR DESCRIPTION
**Pull request by danielnorkin**
# Description

The purpose of this document is to serve as a reference to a methodology that can be used to communicate NFRs to stakeholders. 

## Related Issue

This isn't related to any issue.

## Motivation and Context

This was methodology was discussed during a SSC meeting and was one of the action items. It could help in beginning to address elements of the business value that baselining provides.

## How Has This Been Tested

n/a

## Types of changes

adding a new .md in the docs folder

## Checklist

n/a

### CODE REVIEW

**Comment by skarred14** [comment]
was this meant to be done in gitbook and should be updated in docs/ on the repo as well? @humbitious @danielnorkin

File: docs/Non-Functional Requirements Questionnaire.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+- [ ] **Confidential Code Problem**: You can’t just hide the data with something like ZK-SNARKS and think that everything is ok from a corporate perspective. Many business agreements are embodied in code... business logic. If a machine can execute a smart contract, it can decompile and look at the logic, and that can leak sensitive info.
+- [ ] **Non-Rational (Power) Problems**: IBM, Oracle, SAP and others are telling me what to use. They have intimated that if I don't promote their version of private, permissioned blockchain, they might be less interested in selling my offerings to their customers, and we rely on that.  [Or any number of other reasons why one might not choose Mainnet, even if one believed it was a good technical fit.]
+- [ ] **Emotional**: Bitcoin and Ethereum are for criminals and terrorists.  I don't want to be associated with that, and I'm afraid of what might happen if government cracked down on public blockchains.
+
+## General
+
+### Which vertical or horizontal is your use case in?
```
Comment (breakpointer) [inline docs/Non-Functional Requirements Questionnaire.md:41]: Suggestion: make this a free-form text entry or provide an option for 'other'.

**Comment by danielnorkin** [comment]
The SSC discussed adding a new branch as a NFR questionnaire "sandbox"
before it got added to master. But it should be an .md in the repo where I
placed and not in gitbook so others can potentially edit.

**Comment by breakpointer** [comment]
I'm closing this because I think (correct me if I'm wrong) that now we have the `specifications` branch where this would live? Re-open if you still want this merged in

**Pull request closed without merging**